src/pages/Staking/index.tsx

```diff
@@ -727,7 +751,8 @@ const Staking = (): JSX.Element => {
     votingBalanceGreaterThanZero === undefined ||
     remainingBlockNumbersToUnstake === undefined ||
     availableLockTime === undefined ||
-    availableLockTime <= 0;
+    availableLockTime <= 0 ||
+    unlockFirst;
```

Comment:
The lock time field is disabled when unlock is required first in order to stake again. This way, I prevent any potential bugs from happening in that field especially validation.